ci_framework/roles/env_op_images/README.md

```diff
@@ -0,0 +1,14 @@
+# env_op_images
+A role to gather the container images used in the openstack deployment with specific tags. The file created as part of this role can be used with override_op_image role during deployment to recreate the openstack environment
+
+## Parameters
+* `cifmw_env_op_images_dir`: (String) Directory where the operator_images.yaml will be stored. Defaults to `~/ci-framework-data/artifacts`
+
```

Comment:
blank line - will break list during the doc generation

Reply:
Updated